pyannote/audio/applications/multilabel.py

```diff
@@ -0,0 +1,584 @@
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# AUTHORS
+# Marvin Lavechin - marvinlavechin@gmail.com
+
+"""
+Multi-label classifier
+
+Usage: 
+  pyannote-multilabel train [options] <experiment_dir> <database.task.protocol>
+  pyannote-multilabel validate [options] [--every=<epoch> --chronological --precision=<precision> --use_der] <label> <train_dir> <database.task.protocol>
+  pyannote-multilabel apply [options] [--step=<step>] <model.pt> <database.task.protocol> <output_dir>
+  pyannote-multilabel -h | --help
+  pyannote-multilabel --version
+
+Common options: 
+  <database.task.protocol>   Experimental protocol (e.g. "BabyTrain.SpeakerRole.JSALT")
+  --database=<database.yml>        Path to database configuration file.
+  --subset=<subset>          Set subset (train|developement|test).
+                             Defaults to "train" in "train" mode. Defaults to
+                             "development" in "validate" mode. Defaults to all subsets in
+                             "apply" mode.
+  --gpu                      Run on GPUs. Defaults to using CPUs.
+  --batch=<size>             Set batch size. Has no effect in "train" mode.
+                             [default:  32]
+  --from=<epoch>             Start {train|validat}ing at epoch <epoch>. Has no
+                             effect in "apply" mode. [default:  0]
+  --to=<epochs>              End {train|validat}ing at epoch <epoch>.
+                             Defaults to keep going forever.
+"train" mode: 
+  <experiment_dir>           Set experiment root directory. This script expects
+                             a configuration file called "config.yml" to live
+                             in this directory. See "Configuration file"
+                             section below for more details.
+
+"validation" mode: 
+  --every=<epoch>            Validate model every <epoch> epochs [default:  1].
+  --chronological            Force validation in chronological order.
+  --parallel=<n_jobs>        Process <n_jobs> files in parallel. Defaults to
+                             using all CPUs.
+  <label>                    Label to predict (KCHI, CHI, FEM, MAL or speech).
+                             If options overlap and speech have been activated during training, one
+                             can also to validate on the classes (SPEECH, OVERLAP).
+  <train_dir>                Path to the directory containing pre-trained
+                             models (i.e. the output of "train" mode).
+  --precision=<precision>    Target detection precision [default:  0.8].
+  --use_der                  Indicates if the Detection Error Rate should be used for validating the mode.
```

Comment:
Can we rename this to `--detection`?
This is to make it consistent with the [`--diarization`](https://github.com/pyannote/pyannote-audio/blob/a9891c59105c7e0002bd21f5d53b89db2aa65a52/pyannote/audio/applications/change_detection.py#L69) option in speaker change detection.